easybuild/easyconfigs/g/GROMACS/GROMACS-2016.4_fix_search_for_nvml_include.patch

```diff
@@ -0,0 +1,12 @@
+diff -ru gromacs-2016.4.orig/cmake/FindNVML.cmake gromacs-2016.4/cmake/FindNVML.cmake
```

Comment:
@akesandgren Please include short patch description + mention author or reference to source of patch

Reply:
It contains a description and author, me.
Or do you want it to explicitly say "Author: ", instead of just my name?

Reply:
No, this is fine.